src/hammer-vlsi/defaults.yml

```diff
@@ -970,6 +978,39 @@ power.submit:
   settings: "vlsi.submit.settings"
   settings_meta: lazycrossref
 
+# Specific inputs for the formal tool.
+# These may vary based on the specific tool plugin
+formal.inputs:
+  # Check type to run. Support varies based on the specific tool plugin.
+  # Potential options (see tool plugin defaults.yml for supported checks): 
+  # - lec: logical equivalence checking
+  # - power: power intent (CPF/UPF) and structure checking
```

Comment:
Can we indicate here at the end of each line whether they are currently supported or not